emails/views.py

```diff
@@ -301,6 +301,7 @@ def _get_relay_recipient_from_message_json(message_json):
 
 def _sns_message(message_json):
     incr_if_enabled('sns_inbound_Notification_Received', 1)
+    _record_receipt_verdicts(message_json)
```

Comment:
@groovecoder should this happen after the `commonHeaders` check to ensure it's a well formatted message before trying to parse the JSON payload?